salvo/src/lib/source_tree.py

```diff
@@ -164,8 +166,31 @@ def get_source_directory(self) -> str:
     """
     self._validate()
 
-    return  self._source_repo.source_path \
-                 if self._source_repo.source_path else self._tempdir
+    return self._build_dir.name
+
+  def copy_source_directory(self) -> bool:
```

Comment:
Are these changes also relevant to #90? If not, can we move them to a separate PR to keep this one focused?

Reply:
I can pull this out of this PR